src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift

```diff
@@ -149,12 +153,29 @@ class ENAUITests_01b_Statistics: CWATestCase {
 			XCTAssert(self.app.staticTexts[infectionsTitle].waitForExistence(timeout: .medium))
 			snapshot("statistics_7Day_incidence")
 			app.staticTexts[infectionsTitle].swipeLeft()
+			
 			XCTAssert(self.app.staticTexts[keySubmissionsTitle].waitForExistence(timeout: .medium))
 			snapshot("statistics_key_submissions")
 			app.staticTexts[keySubmissionsTitle].swipeLeft()
+			
 			XCTAssert(self.app.staticTexts[reproductionNumberTitle].waitForExistence(timeout: .medium))
 			snapshot("statistics_7Day_rvalue")
-			cardReproductionNumberOpenInfoScreen(reproductionNumberTitle)
+			app.staticTexts[reproductionNumberTitle].swipeLeft()
+			
+			XCTAssert(self.app.staticTexts[atLeastOneVaccinationTitle].waitForExistence(timeout: .medium))
+			snapshot("statistics_at_lease_one_vaccination")
```

Comment:
Don't know if it matters, but should it not mean: "statistics_at_least_one_vaccination"?

Reply:
correct, i will fix it, thanks!